unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,575 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+######################################
+# This set of tests is valid for any device that supports the HDR feature #
+######################################
+
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy import test, log
+import time
+
+# HDR CONFIGURATION TESTS
+with test.closure("HDR Config - default config"):
+    device = test.find_first_device_or_exit()
+    depth_sensor = device.first_depth_sensor()
+
+    if depth_sensor and depth_sensor.supports(rs.option.hdr_enabled):
+        exposure_range = depth_sensor.get_option_range(rs.option.exposure)
+        gain_range = depth_sensor.get_option_range(rs.option.gain)
+
+        depth_sensor.set_option(rs.option.hdr_enabled, 1)
+        test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+        depth_sensor.set_option(rs.option.sequence_id, 1)
+        test.check(depth_sensor.get_option(rs.option.sequence_id) == 1)
+        exp = depth_sensor.get_option(rs.option.exposure)
+        test.check(depth_sensor.get_option(rs.option.exposure) == exposure_range.default - 1000)  # w/a
+        test.check(depth_sensor.get_option(rs.option.gain) == gain_range.default)
+
+        depth_sensor.set_option(rs.option.sequence_id, 2)
+        test.check(depth_sensor.get_option(rs.option.sequence_id) == 2)
+        test.check(depth_sensor.get_option(rs.option.exposure) == exposure_range.min)
+        test.check(depth_sensor.get_option(rs.option.gain) == gain_range.min)
+
+        depth_sensor.set_option(rs.option.hdr_enabled, 0)
+        test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 0)
+
+        device.hardware_reset()
+        time.sleep(5)
```

Comment:
1. isn't it long? 5 seconds.
2. I see these lines are between each test, please make a function for it